src/arrays/practice/1.js

```diff
@@ -1,9 +1,35 @@
-function arrayManipulations() {
-  // your code here
-  let arr = [1,2,3];
-  arr.filter((elem, i) => {
-      console.log(elem);
-  });
+function arr1(arr) {
+    let LastItem = arr[arr.length - 1];
```

Comment:
Keep variables in camel case. LastItem -> lastItem

Reply:
done